### ISSUE TITLE
weird behaviour (desync/stop, then sped up video) when using seek forward/backward with external audio track (.mka)

### ISSUE THREAD

**Issue by geextahslex** [mpv-player/mpv#14729]
### mpv Information

```bash
mpv 0.38.0 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Apr 20 2024 00:22:40
libplacebo version: v7.349.0 (v6.338.0-124-gd0eb997-dirty)
FFmpeg version: N-114897-gbba996d6c
FFmpeg library versions:
   libavutil       59.15.100
   libavcodec      61.5.103
   libavformat     61.3.100
   libswscale      8.2.100
   libavfilter     10.2.101
   libswresample   5.2.100
```

### Other Information

```text
Windows version: Windows 10 22H2 Build 19045.4170)
GPU model, driver and version: intel HD Graphics 4600, driver: 20.19.15.5171
Source of mpv: https://sourceforge.net/projects/mpv-player-windows/files/
Introduced in version: v38.0
```

### Reproduction Steps

watch video with an external .mka audio track and seek forward or backward

### Expected Behavior

seek works smooth like with an internal (inside the .mkv) audio track

### Actual Behavior

when using seek forward or backward. The video will stop for a moment and then speed up to catch up to the part where the audio is already playing

### Log File

gpu-debug
[output.txt](https://github.com/user-attachments/files/16732293/output.txt)
normal logfile
[output.txt](https://github.com/user-attachments/files/16732347/output.txt)

### I carefully read all instruction and confirm that I did the following:

- [X] I tested with the latest mpv version to validate that the issue is not already fixed.
- [X] I provided all required information including system and mpv version.
- [X] I produced the log file with the exact same set of files, parameters, and conditions used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
- [X] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [X] I attached the full, untruncated log file.
- [X] I attached the backtrace in the case of a crash.

**Comment by Dudemanguy** [comment]
Pretty sure this was fixed by #14399.

**Comment by geextahslex** [comment]
yes the video stopping and speeding up was solved. But still if you seek backwards it takes about 2 seconds for the audio to work. So it still handles external audio worse than native

**Comment by Dudemanguy** [comment]
I just gave it a try myself with a webm + mka and was not able to reproduce.

**Comment by geextahslex** [comment]
how long was your video? I have this issue with episodes that are about 40-50min long 1080p

**Comment by Dudemanguy** [comment]
Took a 2 hour movie, extracted the audio, played it with `--audio-file` and there were no problems when seeking backwards. If you think this is a problem, please open a new issue preferably with sample files that reproduce the issue. Might be AO-specific or something. I'm on ALSA.

**Comment by geextahslex** [comment]
I guess I should have mentioned that by "seek backwards" I mean the default keybind `#LEFT  seek -5                          # seek 5 seconds backward`
I don't mean clicking on the timebar, that works normally ^^'

**Comment by Dudemanguy** [comment]
I was also seeking backwards with keybinds in my testing.

**Comment by hooke007** [comment]
https://github.com/mpv-player/mpv/issues/10887
It was never fixed for me.

**Comment by Dudemanguy** [comment]
Huh, guess I'll reopen then.

**Comment by geextahslex** [comment]
okay I updated the issue description and title, and took the video part out. So it's now only about audio

**Comment by mirai-ux** [comment]
same thing happening to me
```
mpv v0.38.0-dirty Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Jul  3 2024 05:59:22
libplacebo version: v7.349.0
FFmpeg version: n7.0.2
FFmpeg library versions:
   libavutil       59.8.100
   libavcodec      61.3.100
   libavformat     61.1.100
   libswscale      8.1.100
   libavfilter     10.1.100
   libswresample   5.1.100
```
running EndeavourOS on kernel version 6.9.2-arch1-1.1
